payments/swish/direct.md

```diff
@@ -185,7 +186,9 @@ Content-Type: application/json
 |          | └─➔&nbsp;`orderReference`    | `String(50)`  | The order reference should reflect the order reference found in the merchant's systems.                                                                                                                                                                                                            |
 |          | └─➔&nbsp;`subsite`           | `String(40)`  | The subsite field can be used to perform split settlement on the payment. The subsites must be resolved with Swedbank Pay reconciliation before being used.                                                                                                                                        |
 |          | └─➔&nbsp;`msisdn`            | `String`      | Number will be prefilled on payment page, if valid.                                                                                                                                                                                                                                                |
-|          | └─➔&nbsp;`ecomOnlyEnabled`   | `boolean`     | `true` if to only enable Swish on web based transactions.; otherwise `false` to also enable Swish transactions via in-app payments.                                                                                                                                                                |
+|          | └─➔&nbsp;`ecomOnlyEnabled`   | `boolean`     | `true` if to only enable Swish on web based transactions.; otherwise `false` to also enable Swish transactions via in-app payments.
```

Comment:
Since the variables has been moved into a new object this new object has to be described and added to this table.